### ISSUE TITLE
4.10 Undiscovered Map Marker Fomod

### ISSUE THREAD

**Issue by LivelyDismay** [foreverphoenix/the-phoenix-flavour#12]
4.10 - Undiscovered Means Unknown

One of the options of the fomod is:
Which icon do you wish to use for undiscovered locations?
Nothing

I understand that is the most immersive option after all, but has an odd side effect: when you are given a map marker, it will show as "blank", giving the same effect as not having the marker at all.

Just some food for thought, since almost all options in UMU fomod are personal choices anyway.

**Comment by caiobraz** [comment]
Hey there, just for the record, i'm Sr. Kaio from the discord server, which pointed out this issue. :)

I figured out a way to keep the immersion from blank markers, without losing the markers on the map.

The great immersion breaker is the markers appearing on the compass, revealing places that we're not supposed to know. Those markers do not appear on the map, unless given by a book, quest or something else.

So, I tried just changing the first option in the fomod, installing UMU just for the compass with blank markers. They will not appear on the compass, but will show normally on the map, giving us the immersion without losing the proper map markers.
